macros/gemPlotter.py

```diff
@@ -1,18 +1,288 @@
 #!/bin/env python
 
+r"""
```

Comment:
what does this stray `r` character do here?

Reply:
It's a marker for a "raw" string, just like "u" is for Unicode. Escape sequences in raw strings are not interpreted (so `r"\" == "\\"`)

Reply:
Could you add a comment just above describing this to make sure it's not overlooked and removed by someone who doesn't recognise it (e.g. someone like me :D )

Reply:
I'm generally opposed to commenting things for the sole purpose of telling another developer what the correct syntax is...

Reply:
It does deserve an entry in the yet-to-be-written developer docs, though